day_6/exercises/burrito.rb

```diff
@@ -11,9 +11,27 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(topping)
+    @toppings = toppings << topping
+  end
+
+  def remove_topping(topping)
```

Comment:
How would you change this method so that it is removing a topping from the ingredients array?